### PR TITLE
Task/disable authorization

### PR DESCRIPTION
**Pull request by dmoranj**
Add an authentication-only mode, with or without header validation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,13 +108,15 @@ bin/pep-proxy.js
...
+Note that, in order for a request to be validated through the Access Control, it's mandatory that it contains the all of following headers:
 * **x-auth-token**: should contain a valid user token, generated by the IDM.
 * **fiware-service**: should contain the name of a service created in the IDM and the user must have access to it.
 * **fiware-servicepath**: should contain a route to a subservice, begining with a slash '/' symbol.
 
 This headers are used by the IDM and Access Control systems to make the decisions about the request, so if any of them are missing, the request will not progress any further, and will be rejected with a 400 HTTP error code. For a detailed explanation of the returned errors, please refer to the [API Error codes](errorcodes.md).
 
+The proxy can also work in an authentication-only mode (usin the `config.access.disable` flag), in which case the `fiware-service` and `fiware-servicepath` headers can be marked as optional, or checked anyway for validity (controlled by the `config.authentication.checkHeaders` flag). Header checking cannot be disabled for PEP Proxies performing validation processes.
```
Comment (fgalan) [inline README.md]: As discussed by skype "validation" is a bit ambiguous and we should use a more precise wording.

In addition, I'd suggest to change `config.access to `config.authorization`, to keep it aligned with `config.authentication`.

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -8,4 +8,5 @@
 - Remove specific setup for RPM installation in Context Broker uses (#159)
 - Add administration port with Version resource (#164)
 - Check all the mandatory headers before processing the request (#165) (#110)
-- Group and redefine error codes.
\ No newline at end of file
+- Group and redefine error codes.
+- Add unvalidated execution mode for the PEP Proxy (#173)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: "unvalidated"... should be changed to make it clearer.

Comment (dmoranj) [inline CHANGES_NEXT_RELEASE]: Fixed in  14dba92

File: README.md (hunk 1 above)
Comment (dmoranj) [inline README.md]: Not exactly the same case. In this case, we "check the headers for validity" not for authorization (we check that headers: exist, are not empty (as it is said the last bug) and correspond to a user's project). I'll enumerate this checks, anyway.

**Comment by jcalderin** [comment]
LGTM (Passing the ball to @fgalan to merge + delete)

File: README.md (hunk 1 above)
Comment (fgalan) [inline README.md]: Partly deferred to tech debt issue at https://github.com/telefonicaid/fiware-orion-pep/issues/177

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**